201-vmss-windows-customimage/azuredeploy.json

```diff
@@ -0,0 +1,290 @@
+{
+	"$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"vmSSName": {
+			"type": "string",
+			"metadata": {
+				"description": "The Name of the VM Scale Set"
+			}
+		},
+		"instanceCount": {
+			"type": "int",
+			"metadata": {
+				"description": "Number of VM instances to create in the scale set"
+			}
+		},
+		"vmSize": {
+			"type": "string",
+			"allowedValues": [
+				"Standard_D1",
+				"Standard_DS1",
+				"Standard_D2",
+				"Standard_DS2",
+				"Standard_D3",
+				"Standard_DS3",
+				"Standard_D4",
+				"Standard_DS4",
+				"Standard_D11",
+				"Standard_DS11",
+				"Standard_D12",
+				"Standard_DS12",
+				"Standard_D13",
+				"Standard_DS13",
+				"Standard_D14",
+				"Standard_DS14"
+			],
+			"metadata": {
+				"description": "The size of the VM instances Created"
+			}
+		},
+		"dnsNamePrefix": {
+			"type": "string",
+			"metadata": {
+				"description": "The Prefix for the DNS name of the new IP Address created"
+			}
+		},
+		"adminUsername": {
+			"type": "string",
+			"metadata": {
+				"description": "The Username of the admininstrative user for each VM instance created"
+			}
+		},
+		"adminPassword": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The Password of the admininstrative user for each VM instance created"
+			}
+		},
+		"location": {
+			"type": "string",
+			"metadata": {
+				"description": "Deployment location"
+			}
+		},
+		"imageSource": {
```

Comment:
I assume this is a vhd Uri, if you it'd be better to call it sourceImageVhdUri or soemthing that makes it clear from the name